examples/with-react-relay-network-modern/next.config.js

```diff
@@ -0,0 +1,21 @@
+require('dotenv').config()
+
+const path = require('path')
+const webpack = require('webpack')
+const Dotenv = require('dotenv-webpack')
+
+module.exports = {
+  webpack: (config, { isServer }) => {
+    config.plugins.push(
+      new webpack.DefinePlugin({
+        __IS_SERVER__: isServer.toString()
+      }),
+      new Dotenv({
+        path: path.join(__dirname, '.env'),
+        systemvars: true
+      })
+    )
+
+    return config
+  }
+}
```

Comment:
`next.config.js` is not needed for this example. Instead of `__IS_SERVER__` use `!process.browser`, this is automatically defined by Next.js. DotEnv support is out of scope for this example.

Reply:
Removed.